lib/Target/TfheRustBool/TfheRustBoolEmitter.cpp

```diff
@@ -215,9 +245,25 @@ void TfheRustBoolEmitter::emitReferenceConversion(Value value) {
 LogicalResult TfheRustBoolEmitter::printSksMethod(
     ::mlir::Value result, ::mlir::Value sks, ::mlir::ValueRange nonSksOperands,
     std::string_view op, SmallVector<std::string> operandTypes) {
-  if (isa<TensorType>(nonSksOperands[0].getType())) {
+  // ToDo: Check again the enum definition
+  if (packedAPI && (op == "and" || op == "nand" || op == "or" || op == "nor" ||
```

Comment:
could you please add a small comment describing the output expect this branch to take? i finally realized it was writing normal boolean operations using the packed API haha

Something like "If using the packed API, then emit single boolean operations as ...."